examples/testplans/release.json

```diff
@@ -12,6 +12,9 @@
 	{"name": "Avocado RPM build",
 	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
 
+	{"name": "Avocado RPM lint",
+	 "description": "With the built RPMs, check if there are no packaging errors or warnings by running: `$ rpmlint BUILD/RPM/avocado-x.y.z-r.distro.noarch.rpm`.  Expected result: 1 packages and 0 specfiles checked; 0 errors, 0 warnings."},
```

Comment:
What is the difference between `spec` file and `rpm` file check? Shouldn't we check both? (I noticed it says `1 packages and 0 specfiles checked; 0 errors, 0 warnings.` and when I try `rpmlint avocado.spec` it also reports no errors.

Also how about adding `make rpmcheck` target?